src/ChunkingContext.jl

```diff
@@ -0,0 +1,82 @@
+# When splitting the work among multiple tasks, each task should have at least this many bytes of input
+# This is to avoid having too many tasks with too little work to do.
```

Comment:
..... what happens if the buffer is smaller than num-tasks * this constant? Do we still split it, but put the extra bytes at the end of one of the tasks? Or just don't split it at all?
I think this comment should say that.